lib/get-repositories.js

```diff
@@ -1,9 +1,24 @@
 const fs = require('fs');
+const { isArray } = require('lodash');
```

Comment:
Worth making an issue to not depend on all of `lodash`.

https://youmightnotneed.com/lodash/

Reply:
I've used Array.isArray 👍